src/extraction/fathom/ruleset_factory.js

```diff
@@ -237,7 +236,9 @@ export default class RulesetFactory {
 
   hasBackgroundImage(fnode) {
     const bgImage = getComputedStyle(fnode.element)['background-image'];
-    return !!bgImage && bgImage !== 'none';
+    return bgImage && bgImage.startsWith('url("') && bgImage.endsWith('")');
+    // The app doesn't yet know how to deal with non-url() types of background
+    // images.
```

Comment:
I don't think we need to change it but comments after the return statement are messin with my brain

Reply:
Yeah, I thought it was a little funny too, but it seemed like where they should go in the flow of thought. :-)